srcpkgs/audit/template

```diff
@@ -15,13 +15,21 @@ license="GPL-2.0-or-later, LGPL-2.0-or-later"
 homepage="https://people.redhat.com/sgrubb/audit"
 changelog="https://raw.githubusercontent.com/linux-audit/audit-userspace/master/ChangeLog"
 distfiles="${homepage}/${pkgname}-${version}.tar.gz"
-checksum=23777e1dc9a80a2ee06a4d442a6a0a9bcbf1ae7ee4b5738a220ff619738cc904
+checksum=220a78cf35e6a7c57f42fcfc42b759692641c7e8e0e39c9844fbc0aa51783631
 
 case "$XBPS_TARGET_MACHINE" in
 	*-musl) configure_args+=" --disable-zos-remote" ;;
 	*) ;;
 esac
 
+post_patch() {
+	# See https://github.com/linux-audit/audit-userspace/issues/381
+	case "$XBPS_TARGET_MACHINE" in
+		*-musl) patch -Np1 -i "${FILESDIR}/musl.patch" ;;
```

Comment:
if this patch works on glibc, don't apply it conditionally. conditional patches are icky